prometheus/generic/setup-bind-exporter/tasks/docker.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Run the bind_exporter
+  docker_container:
+    name: bind_exporter
+    image: "{{ bind_exporter_image }}:{{ bind_exporter_image_version }}"
+    pid_mode: "host"
```

Comment:
Is this required? Do we need to share the host's process namespace?

Reply:
Makes sense. Removed the pid_mode: line from the task